akka-remote/src/main/scala/akka/remote/transport/ThrottlerTransportAdapter.scala

```diff
@@ -474,7 +474,7 @@ private[transport] class ThrottledAssociation(
 
   when(WaitOrigin) {
     case Event(InboundPayload(p), ExposedHandle(exposedHandle)) =>
-      throttledMessages = throttledMessages.enqueue(p)
+      throttledMessages = throttledMessages.appended(p: ByteString)
```

Comment:
apparently `value appended is not a member of scala.collection.immutable.Queue[akka.util.ByteString]` on Scala 2.12, yet :/

Reply:
Dang, and on Scala 3 enqueue binds to the wrong method here (the deprecated one taking Iterable or something like that). I guess that could potentially be a compiler bug though, choosing the wrong overload.